app/src/main/java/net/squanchy/service/repository/injection/RepositoryModule.kt

```diff
@@ -34,6 +33,7 @@ class RepositoryModule {
     }
 
     @Provides
-    @ApplicationLifecycle
-    internal fun filterScheduleRepository(): FilterScheduleRepository = LocalFilterScheduleRepository()
+    internal fun tracksRepository(dbService: FirestoreDbService): TracksRepository {
```

Comment:
we can have a direct return here (with the equals)

Reply:
Yup we can bdc34e580e25f0402f732935c42fc51e221e537b